lib/elix/responders/gif_me.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Elix.Responders.GifMe do
+  @moduledoc """
+  Searches the Giphy public API and return the URL
+  of a GIF matching the search term.
+  """
+
+  use Hedwig.Responder
+
+  @base_url "http://api.giphy.com/v1/gifs/search"
+  @api_key "dc6zaTOxFJmzC" # Giphy’s public beta API key
+  @sample_size 10
+  @api_client Application.get_env(:elix, :api_client)
+
+  @usage """
+  gif me <term> - Replies with a GIF URL matching the term
+  """
+  hear ~r/gif me (.+)/i, msg do
```

Comment:
Worth adding in the [`from_self`](https://github.com/indyelixir/elix/commit/b10e628ca07fcc0406c2d0eb845f8b18d375fcdc#diff-ffe52e6014b7809b6bfc8713165e838aR33) bits here? Seems unlikely elix will be talking to itself about gifs, but worth a thought. I relaly wonder why `ignore_self`  was removed